source

```
@@ -12491,6 +12493,15 @@ interface <dfn>HTMLLinkElement</dfn> : <span>HTMLElement</span> {
   tokens</span> must only include the tokens from this list that the user agent implements the
   processing model for.</p>
 
+  <p>The <dfn><code data-x="attr-link-as">as</code></dfn> attribute specifies the <span
+  data-x="concept-request-destination">destination</span> for a preloaded response from the resource
+  given by the <code data-x="attr-link-href">href</code> attribute. The <code
+  data-x="attr-link-as">as</code> attribute must not be specified on <code>link</code> elements that
+  do not have a <code data-x="attr-link-rel">rel</code> attribute that contains the <code
+  data-x="rel-preload">preload</code> keyword.<span w-nodev> User agents must implement the
+  processing model of the <code data-x="attr-link-as">as</code> attribute described in
+  <cite>Preload</cite>. <ref spec=PRELOAD></span></p>
```

Should add a conformance requirement that it be a valid destination.

Should it be an enumerated attribute? That would mean that `linkEl.as = "asdf"` falls back to the invalid value default (presumably `""`), so afterward `linkEl.as === ""`. This can be useful for feature detection.